### DISCUSSION TITLE
URL Cleaner pattern

### DISCUSSION THREAD

**Discussion by MicroBeast** [TrianguloY/URLCheck#373] [Q&A] [answered]
Does the upstream filter list last update really Sep 1, 2022?

![Screenshot_20240803_180220](https://github.com/user-attachments/assets/d28b2e03-71fa-4225-9776-13bac3416b45)

On https://github.com/ClearURLs/Rules seem last update was 4 months ago (at time of this writing).

Is this Cosmetic issue?

**Comment by TrianguloY** [comment, marked as answer]
I remember thinking about this in the past...it seems I totally forgot about it.

The internal catalog is being updated automatically, but the time is not. I guess I need to update it too.

Technical details: there is a Github action that checks the catalog and creates a pr to update it whenever a new version exists. That action doesn't change the "last update" time from https://github.com/TrianguloY/UrlChecker/blob/master/app%2Fsrc%2Fmain%2Fjava%2Fcom%2Ftrianguloy%2Furlchecker%2Fmodules%2Fcompanions%2FClearUrlCatalog.java and it should.
If you ever update the catalog (when a new version is published and the app still doesn't have it) the time will be correctly updated. Until then, that hardcoded outdated time will be shown.

**Comment by MicroBeast** [comment]
It's basically just really cosmetic then. I was just wondering, hence I didn't post on Issue trackers.

Thx.
